settings/prod.py

```diff
@@ -24,14 +24,19 @@
 
 INSTALLED_APPS += ('storages',)  # noqa
 
+AWS_STORAGE_BUCKET_NAME = "evalai"
```

Comment:
So, the bucket name is `evalai` and static files will be stored in `evalai/static` and media files will be stored in `evalai/media`. Does that sound good?